app/page.tsx

```
@@ -129,206 +135,293 @@
+          <div className="space-y-4">
+            {userProfile.education.map((edu, index) =>
+              <div key={index} className="cv-card education-item">
+                <div className="flex flex-col sm:flex-row sm:justify-between sm:items-start mb-2">
+                  <h3 className="font-semibold text-slate-800">
+                    {edu.institution}
+                  </h3>
+                  <span className="cv-badge bg-slate-100 text-slate-500">
+                    {edu.dateRange}
+                  </span>
+                </div>
+                <p className="text-slate-700" dangerouslySetInnerHTML={{ __html: sanitize(`${edu.degree}, ${edu.field}`) }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)